src/components/dashboard/SnapshotWidget/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { Box, Card, CardContent, Chip, Link, SvgIcon, Typography, Skeleton } from '@mui/material'
+
+import LinkIconBold from '@/public/images/sidebar/link-bold.svg'
+import useSafeSnapshot from '@/components/dashboard/SnapshotWidget/useSafeSnapshot'
+import type { SnapshotProposal } from '@/components/dashboard/SnapshotWidget/useSafeSnapshot'
+
+import css from './styles.module.css'
+
+// These colors remain the same regardless of dark/light mode
+const STATUS_TEXT_COLOR = '#FFF'
+const CLOSED_STATUS_COLOR = '#743EE4'
+
+export const _getProposalNumber = (title: string): string => {
+  // Find anything that matches "SEP #n"
+  const SEP_REGEX = /SEP\s\#\d+/g
+  return title.match(SEP_REGEX)?.[0] || ''
+}
+
+export const _getProposalTitle = (title: string): string => {
+  // Find anything after "] " or ": "
+  const TITLE_REGEX = /(\]|\:) (.*)/
+  return title.match(TITLE_REGEX)?.at(-1) || ''
+}
+
+const LinkIcon = () => <SvgIcon component={LinkIconBold} inheritViewBox fontSize="small" className={css.icon} />
+
+const SnapshotProposals = ({ proposals }: { proposals: SnapshotProposal[] }) => (
+  <>
+    {proposals?.map((proposal) => (
+      <a
+        href={`${SNAPSHOT_LINK}/proposal/${proposal.id}`}
+        key={proposal.id}
+        target="_blank"
+        rel="noopener noreferrer"
+        className={css.proposal}
+      >
+        <Box gridArea="number">
+          <Chip label={_getProposalNumber(proposal.title)} className={css.number} />
+        </Box>
+        <Box gridArea="title" className={css.title}>
+          <Typography>{_getProposalTitle(proposal.title)}</Typography>
+        </Box>
+        <Box gridArea="status">
+          <Chip
+            label={proposal.state}
+            sx={{
+              color: STATUS_TEXT_COLOR,
+              backgroundColor: proposal.state === 'active' ? 'success.main' : CLOSED_STATUS_COLOR,
+            }}
+            className={css.status}
+          />
+        </Box>
+        <Box gridArea="link">
+          <LinkIcon />
+        </Box>
+      </a>
+    ))}
+  </>
+)
+
+const SNAPSHOT_LINK = 'https://snapshot.org/#/safe.eth'
+const FORUM_LINK = 'https://forum.gnosis-safe.io'
+
+const SnapshotWidget = () => {
+  const SKELETON_NUMBER = 2
```

Comment:
We should have a different constant name like "DISPLAYED_PROPOSALS" and go with 3 as in the mockups.

Reply:
I've adjusted it to `3` and now pass it to `useSnapshot`.